user-api/src/main/java/com/example/auction/user/api/User.java

```diff
@@ -11,12 +11,14 @@
 public final class User {
 
     private final UUID id;
+    private final UUID creatorId;
     private final String name;
     private final String email;
 
     @JsonCreator
-    public User(@JsonProperty("id") UUID id, @JsonProperty("name") String name, @JsonProperty("email") String email) {
+    public User(@JsonProperty("id") UUID id, @JsonProperty("creatorId") UUID creatorId , @JsonProperty("name") String name, @JsonProperty("email") String email) {
```

Comment:
I think we want it to be a timestamp (maybe called `createdAt`) rather than a UUID.